soc/arm/st_stm32/stm32wb/power.c

```diff
@@ -117,8 +117,7 @@ static int stm32_power_init(struct device *dev)
 
 	ret = irq_lock();
 
-	/* enable Power clock */
-	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
+	/* no specific Power clock to enable */
```

Comment:
Remove the unnecessary IRQ lock:
```
static int stm32_power_init(struct device *dev)
{
        ARG_UNUSED(dev);

        /* no specific Power clock to enable */
        return 0;
}

Reply:
Actually, remove the whole `SYS_INIT`.

Reply:
yes in the end, Removing the stm32_power_init and SYS_INIT